changelog.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v0.13.2 | tbd
+
+### New features
+* Fixed and reintroduced lazy loading for `TimeSeries` data. 
```

Comment:
There needs to be a linebreak between `New features` and the first item here.

Reply:
fixed